static/src/javascripts/projects/common/modules/identity/upsell/store/fetch.js

```diff
@@ -1,50 +0,0 @@
-import {
-    getAllConsents,
-    getAllNewsletters,
-    getSubscribedNewsletters,
-    getUserFromApi,
-} from 'common/modules/identity/api';
-import { UserConsentWithState, EmailConsentWithState } from './types';
-
-const fetchSubscribedConsents = () =>
-    new Promise(accept => {
-        getUserFromApi(user => {
-            if (user && user.consents) {
-                accept(
-                    user.consents
-                        .filter(consent => consent.consented === true)
-                        .map(consent => consent.id)
-                );
-            } else {
-                accept([]);
-            }
-        });
-    });
-
-const fetchUserConsents = Promise.all([
-    fetchSubscribedConsents(),
-    getAllConsents(),
-]).then(([acceptedConsents, allConsents]) =>
-    allConsents.map(
-        consent =>
-            new UserConsentWithState(
-                consent,
-                acceptedConsents.includes(consent.id)
-            )
-    )
-);
-
-const fetchNewsletters = Promise.all([
```

Comment:
this is the offending `const` which was triggered requests to authorised endpoint me/newsletter on IDAPI. This probably stopped working after we stopped giving sessions to users on the registration flow.

It turns out we can remove it.